bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectInverter.xml

```diff
@@ -0,0 +1,115 @@
+			</channel>
+			<channel id="inverter-frequency" typeId="frequency">
+				<label>Inverter Frequency</label>
+				<description>Frequency of the electricity to/from the inverter</description>
+			</channel>
+
+			<channel id="pv1-voltage" typeId="system.electric-voltage">
+				<label>PV 1 Voltage</label>
+				<description>Electric voltage of PV String 1</description>
+			</channel>
+			<channel id="pv2-voltage" typeId="system.electric-voltage">
+				<label>PV 2 Voltage</label>
+				<description>Electric voltage of PV String 2</description>
+			</channel>
+			<channel id="pv1-current" typeId="system.electric-current">
+				<label>PV 1 Current</label>
+				<description>Electric current of PV String 1</description>
+			</channel>
+			<channel id="pv2-current" typeId="system.electric-current">
+				<label>PV 2 Current</label>
+				<description>Electric current of PV String 2</description>
+			</channel>
+			<channel id="pv1-power" typeId="system.electric-power">
+				<label>PV 1 Power</label>
+				<description>Electric power of PV String 1</description>
+			</channel>
+			<channel id="pv2-power" typeId="system.electric-power">
+				<label>PV 2 Power</label>
+				<description>Electric power of PV String 2</description>
+			</channel>
+			<channel id="pv-total-power" typeId="system.electric-power">
+				<label>PV Total Power</label>
+				<description>The sum of PV powers from all strings</description>
+			</channel>
+			<channel id="pv-total-current" typeId="system.electric-current">
+				<label>PV Total Current</label>
+				<description>The sum of PV currents from all strings</description>
+			</channel>
+
+			<channel id="battery-power" typeId="system.electric-power">
+				<label>Battery Power</label>
+				<description>Power to/from the battery</description>
+			</channel>
+			<channel id="battery-current" typeId="system.electric-current">
+				<label>Battery Current</label>
+				<description>Electric current to/from the battery</description>
+			</channel>
+			<channel id="battery-voltage" typeId="system.electric-voltage">
+				<label>Battery Voltage</label>
+				<description>Electric voltage of the battery</description>
+			</channel>
+			<channel id="battery-temperature" typeId="battery-temperature">
+				<label>Battery Temperature</label>
+				<description>Temperature of the battery</description>
+			</channel>
+			<channel id="battery-level" typeId="system.battery-level">
+				<label>Battery Level</label>
+				<description>The battery state of charge in percent</description>
+			</channel>
+
+			<channel id="feed-in-power" typeId="system.electric-power">
+				<label>Feed-in Power</label>
+				<description>Power to/from the electricity network.</description>
+			</channel>
+
+			<channel id="last-update-time" typeId="last-retrieve-time-stamp"/>
+
+			<channel id="raw-data" typeId="raw-data-type">
+				<label>Raw Inverter Data</label>
+				<description>The raw JSON data retrieved from the inverter's Wi-Fi module.</description>
```

Comment:
This overrides the label/description of the channel type and is not consistent with the current label/description of the channel type. You can remove them from here and instead correct the label/description of the channel type.